raiden/transfer/node.py

```diff
@@ -511,7 +512,7 @@ def handle_token_network_action(
     if token_network_state:
         pseudo_random_generator = chain_state.pseudo_random_generator
         iteration = token_network.state_transition(
-            payment_network_identifier=payment_network_id,
+            payment_network_identifier=cast(PaymentNetworkID, payment_network_id),
```

Comment:
don't use `cast` unless the type is a Union type. You can simply do: `PaymentNetworkID(payment_network_id)`.
Use `cast` only when the above does not work, in the case the type you want to cast is a `Union` of types.